bi-connectors/PowerBIConnector/docs/AmazonOpenSearchService.md

```diff
@@ -0,0 +1,66 @@
+#  Connecting Amazon OpenSearch Service to Microsoft Power BI Desktop
+
+>[!Note]
+
+>The following connector article is provided by Amazon, the owner of this connector and a member of the Microsoft Power Query Connector Certification Program. If you have questions regarding the content of this article or have changes you would like to see made to this article, visit the OpenSearch website and use the support channels there.
+
+## Summary
+| Item | Description |
+| ---- | ----------- |
+| Release State | General Availability |
+| Products | Power BI Desktop |
+| Authentication Types Supported | Basic |
+
+## Prerequisites
+* Microsoft Power BI Desktop
+* [OpenSearch](https://docs-beta.opensearch.org/opensearch/install/index/)
+* [OpenSearch SQL ODBC driver](https://docs-beta.opensearch.org/search-plugins/sql/odbc/)
+
+## Capabilities supported
+* Import
+* DirectQuery
+
+## Connect to Amazon OpenSearch Service
+1. Open Power BI Desktop.
+
+2. Click on **Home** > **Get Data** > **More** > **Other**. Select **Amazon OpenSearch Service**. Click on **Connect**.
+
+<img src="img/pbi_select_connector_amz.png" width="500">
+
+3. You will get a warning for using third-party service. Click on **Continue**.
+
+<img src="img/pbi_third_party_warning.png" width="500">
+
+4. Enter server value. Click on **OK**.
+
+<img src="img/pbi_connection_string_options.png" width="500">
+
+5. Select authentication option. Enter credentials if required and click on **Connect**.
+
+<img src="img/pbi_auth.png" width="500">
+
+6. Select required table. Data preview will be loaded.
+
+<img src="img/pbi_data_preview.png">
+
+7. Click on **Load**.
+
+8. Select required columns for creating graph.
+
+<img src="img/pbi_simple_graph.png">
+
+
+## Troubleshooting 
+
+* If you get the following error, please install [OpenSearch SQL ODBC Driver](https://docs-beta.opensearch.org/search-plugins/sql/odbc/).
+
+<img src="img/pbi_error_driver_not_installed.png" width="350">
+
+* If you get the following error,
+
+<img src="img/pbi_error_conn.png" width="350">
+
+1. Check if host and port values are correct.

```

Comment:
Note above that you say the "server" value, but don't mention host and port.

Reply:
Will update this. Thanks

Reply:
Updated the instructions above